src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+const galleryList = document.querySelector('.gallery__list');
+
+galleryList.addEventListener('click', (currentEvent) => {
```

Comment:
Ensure that 'galleryList' is not null before adding an event listener to it. This will prevent potential runtime errors if the element is not found in the DOM.